provisioning/cmd/provisioning-service/README.md

```diff
@@ -17,6 +17,9 @@ configuration:
 - `listen-addr` (optional): the address, in the form `<host>:<port>` the provisioning
   server will be listening on. If not specified, this defaults to
   `localhost:8888`.
+- `protocol` (optional): the protocol that will be used. Must be either "http" or "https". Defaults to "https" if not specified.
+- `cert` (optional): path to the x509 certificate to be used. Must be specified if protocol is "https"
```

Comment:
Should there be default certificates and path to default certificates, when the protocol is omitted and defaults to `https` ?

Reply:
No because there cannot be a sinsible default. Certs is something that _must_ be sourced by the user -- it's part of the deployment, not the code.

Reply:
Also it would be good to refer the new additions and `if required` any mandatory parameters to be supplied or defaults used in the README's associated with P, V, as below..
https://github.com/veraison/services/tree/main/provisioning

Reply:
> No because there cannot be a sensible default. Certs is something that _must_ be sourced by the user -- it's part of the deployment, not the code.

ok, understood, then I think, this does not become an `optional` parameter, as by default this is required, unless the user explicitly sets the protocol parameter to "http" to change the default behaviour. This aspect must be clarified in the README.

Reply:
Yeah good point. This was optional when the default was HTTP.

Reply:
actually the doc already clarifies that